src/app.ts

```diff
@@ -678,7 +689,8 @@ export class App {
     });
   }
 
-  public writeTemplate(filepath: string, template: (data: any) => string, api: gapi.client.discovery.RestDescription) {
+  // tslint:disable-next-line:no-any
+  writeTemplate(filepath: string, template: (data: any) => string, api: gapi.client.discovery.RestDescription) {
```

Comment:
Let's get rid of this `any` by defining `template` this way:

```js
template: RenderFunction
```
and add
```js
import { RenderFunction } from 'dot';
```
at the top of the file.